gmt/tests/test_surface.py

```diff
@@ -0,0 +1,96 @@
+"""
+Tests for surface
+"""
+import os
+
+import pytest
+import xarray as xr
+
+from .. import surface
+from .. import which
+from ..datasets import load_tut_ship
+from ..exceptions import GMTInvalidInput
+from ..helpers import data_kind
+
+TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
+TEMP_GRID = os.path.join(TEST_DATA_DIR, "tmp_grid.nc")
+
+
+def test_surface_input_file():
+    """
+    Run surface by passing in a filename
+    """
+    fname = which("@tut_ship.xyz", download="c")
+    output = surface(data=fname, I="5m", R="245/255/20/30")
```

Comment:
Please use the full argument names (`region` instead of `R`, etc) in the code. We don't want to encourage people to use the single letter arguments.

Reply:
I know we're doing this in other places but I'm trying to change that.

Reply:
Yeah I do prefer the explicit style. I've made the change, but also left in a test for the short form (-I and -R).